doc/development_process/repo_structure.rst

```diff
@@ -0,0 +1,100 @@
+.. _repo-structure:
+
+Repository structure
+####################
+
+Introduction
+************
+
+The Zephyr Project codebase was, for a long time, contained in a single `Git`_
+repository. As the project evolved, it became clear that, :ref:`due to multiple
+reasons <west-why-multi-repo>`, the project's code and supporting files needed
+to be split out into a set of repositories instead.
+
+To that end a new tool, :ref:`west`, was developed and introduced to ease the
+burden of managing multiple Git repositories.
+
+Structure
+*********
+
+The Zephyr Project codebase is composed of several `Git`_ repositories,
+organized in a :ref:`star topology <west-topologies>` as depicted below.
+
+.. figure:: zephyr-repos.png
+    :align: center
+    :alt: Zephyr repository structure
+
+    Zephyr repository structure
+
+There are two types of repositories as shown by the figure:
+
+  * The `manifest repository`_ or also commonly named the "zephyr" repository
+    contains the `west.yml`_ :ref:`manifest file <west-manifests>` that
+    lists all other repositories that the manifest repository depends on.
+  * The project repositories are the ones containing :ref:`west projects
```

Comment:
```
The project repositories are set in the `projects:` part of :file:`west.yml`.
```

They aren't contained there as much as they are the same thing, no?

Reply:
Use double back-ticks around ``projects:`` (not single back ticks)